src/components/register.component.js

```diff
@@ -290,6 +303,7 @@ const Register = () => {
 
                         </div>
 
+                        { registerError ? <p> error  </p> : null }
```

Comment:
Need to handle error message, for example: there is already registered user
And same staff for login

Reply:
done.

Reply:
everything fixed in this branch